### PR TITLE
618 templates action

### PR DESCRIPTION
**Pull request by dgrassellyb**

### CODE REVIEW

File: .github/workflows/check-templates.yml [hunk 1]
```diff
@@ -18,6 +28,29 @@ on:
...
+        "reviewers": ["bundfussr", "manciniedoardo"]
+      },
+      "admiralophtha": {
+        "reviewers": ["manciniedoardo"]
+      },
+      "admiral": {
+        "reviewers": ["manciniedoardo", "bms63"]
+      },
+      "admiralvaccine": {
+        "reviewers": ["bms63", "manciniedoardo"]
```
Comment (bms63) [inline .github/workflows/check-templates.yml:48]: Could a vaccine person be put on this - I don't want to be responsible for that rep.  @manciniedoardo @neetusan

Comment (neetusan) [reply .github/workflows/check-templates.yml:48]: As Sukalpo is the Maintainer, I am checking with Arjun to be one of the reviewers on the vaccine side. We will make updates tomorrow. Thanks Ben for all your help!

Comment (ahasoplakus) [reply .github/workflows/check-templates.yml:48]: Hi @dgrassellyb @manciniedoardo You can put me (*ahasoplakus*) and Arjun (*arjoon-r*) as reviewers for admiralvaccine.

Comment (dgrassellyb) [reply .github/workflows/check-templates.yml]: (updated)

**Pull request merged**